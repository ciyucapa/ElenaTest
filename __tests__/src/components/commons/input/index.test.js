import React from 'react';
import {create} from 'react-test-renderer';

import Input from '../../../../../src/components/commons/input';

describe('Input component', () => {
    test('render input component', () => {
        const tree = create(
            <Input
                type={'text'} 
                value={'value input'} 
                onChange={() => {}}  
                placeholder={'input placeholder'}
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});