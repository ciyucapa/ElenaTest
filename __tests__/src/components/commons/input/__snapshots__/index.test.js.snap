// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component render input component 1`] = `
<input
  onChange={[Function]}
  placeholder="input placeholder"
  type="text"
  value="value input"
/>
`;
